spec/pubid_ieee/identifiers_parsing_spec.rb

```diff
@@ -857,7 +857,7 @@ def context_description(ex)
 
       context "IEEE/ISO/IEC 8802-1Q-2020/Amd31-2021" do
         let(:original) { "IEEE/ISO/IEC 8802-1Q-2020/Amd31-2021" }
-        let(:pubid) { "IEEE/ISO/IEC Std 8802-1Q-2020/Amd31-2021" }
+        let(:pubid) { "IEEE/IEC/ISO Std 8802-1Q-2020/Amd31-2021" }
```

Comment:
Why did this become "IEEE/IEC/ISO"? It is really "IEEE/ISO/IEC":
https://standards.ieee.org/ieee/8802-1Q/10339/
<img width="751" alt="Screenshot 2023-06-23 at 4 09 28 PM" src="https://github.com/metanorma/pubid-ieee/assets/11865/97a42e8a-43b4-40f8-b8c2-b35630076162">

Reply:
@mico thanks for metanorma /pubid-iso#235 , can you help update this PR to merge? Thanks.

Reply:
@ronaldtse I updated `pubid-iso` to preserve publishers order